js_dev/ibm704sim.js

```diff
@@ -91,4 +101,173 @@ function run() {
     return general_memory[12]; // should be 25
 }
 
+function convert_to_binary(number, digits) {
+    result = number.toString(2);
+    length = result.length;
+    for (let i = 0; i < digits - length; i++) {
+        result = "0" + result;
+    }
+    return result.toString();
+}
+
+const no_to_operation_str = {0o601: "STO", 0: "HTR", 0o500: "CLA", 0o400: "ADD", 0b110: "TNX"};
```

Comment:
Can you add a note why some of these are in octal, one in binary, and one in decimal?  Best to choose one base for all.